ddl/ddl_api.go

```diff
@@ -5797,10 +5813,10 @@ func isDroppableColumn(multiSchemaChange bool, tblInfo *model.TableInfo, colName
 	return nil
 }
 
-// validateCommentLength checks comment length of table, column, index and partition.
+// validateIndexCommentLength checks comment length of index
 // If comment length is more than the standard length truncate it
 // and store the comment length upto the standard comment length size.
-func validateCommentLength(vars *variable.SessionVars, indexName string, indexOption *ast.IndexOption) (string, error) {
+func validateIndexCommentLength(vars *variable.SessionVars, indexName string, indexOption *ast.IndexOption) (string, error) {
```

Comment:
I prefer to merge all the situations into one function and pass different parameters.